SPECS/curl.spec

```diff
@@ -322,7 +308,9 @@ rm -f ${RPM_BUILD_ROOT}%{_libdir}/libcurl.la
 # should be fixed in next release https://github.com/curl/curl/pull/12843
 rm -f ${RPM_BUILD_ROOT}%{_mandir}/man1/mk-ca-bundle.1*
 
-%ldconfig_scriptlets -n libcurl
+%post -n libcurl -p /sbin/ldconfig
+
+%postun -n libcurl -p /sbin/ldconfig
```

Comment:
The `%ldconfig_scriptlets` macro is actually available in our build root in 8.3, so we can keep using it here.